build.sbt

```diff
@@ -94,34 +96,39 @@ lazy val commonSettings = gspScalaJsSettings ++ Seq(
 
 lazy val commonLibSettings = gspScalaJsSettings ++ Seq(
   libraryDependencies ++= Seq(
-    "com.github.japgolly.scalajs-react" %%% "core" % scalaJsReact,
-    "com.github.japgolly.scalajs-react" %%% "extra" % scalaJsReact,
-    "com.github.japgolly.scalajs-react" %%% "test" % scalaJsReact % Test,
-    "edu.gemini" %%% "gsp-core-model" % "0.1.8",
-    "edu.gemini" %%% "gpp-ui" % "0.0.3",
-    "org.typelevel" %%% "cats-effect" % "2.1.3",
-    "org.typelevel" %%% "cats-core" % "2.1.1",
-    "org.typelevel" %%% "mouse" % "0.24",
-    "io.chrisdavenport" %%% "log4cats-core" % "1.0.1",
-    "io.chrisdavenport" %%% "log4cats-log4s" % "0.4.0-M1",
-    "io.github.cquiroz.react" %%% "react-semantic-ui" % "0.4.12",
-    "com.github.julien-truffaut" %%% "monocle-core" % "2.0.4",
-    "com.github.julien-truffaut" %%% "monocle-macro" % "2.0.4",
-    "com.rpiaggio" %%% "crystal" % "0.1.6",
-    "com.rpiaggio" %%% "clue-scalajs" % "0.0.6",
-    "io.circe" %%% "circe-generic-extras" % "0.13.0",
-    "io.suzaku" %%% "diode-data" % "1.1.7",
-    "io.suzaku" %%% "diode-react" % "1.1.7.160",
-    "com.lihaoyi" %%% "utest" % "0.7.4" % Test
+    "com.github.japgolly.scalajs-react" %%% "core"                 % scalaJsReact,
+    "com.github.japgolly.scalajs-react" %%% "extra"                % scalaJsReact,
+    "com.github.japgolly.scalajs-react" %%% "test"                 % scalaJsReact % Test,
+    "edu.gemini"                        %%% "gsp-core-model"       % "0.1.8",
+    "edu.gemini"                        %%% "gpp-ui"               % "0.0.3",
+    "org.typelevel"                     %%% "cats-effect"          % "2.1.3",
+    "org.typelevel"                     %%% "cats-core"            % "2.1.1",
+    "org.typelevel"                     %%% "mouse"                % "0.24",
+    "io.chrisdavenport"                 %%% "log4cats-core"        % "1.0.1",
+    "io.chrisdavenport"                 %%% "log4cats-log4s"       % "0.4.0-M1",
+    "io.github.cquiroz.react"           %%% "react-semantic-ui"    % "0.4.12",
+    "com.github.julien-truffaut"        %%% "monocle-core"         % monocle,
+    "com.github.julien-truffaut"        %%% "monocle-macro"        % monocle,
+    "com.rpiaggio"                      %%% "crystal"              % "0.2.0",
+    "com.rpiaggio"                      %%% "clue-scalajs"         % "0.0.6",
+    "io.circe"                          %%% "circe-generic-extras" % "0.13.0",
+    "io.suzaku"                         %%% "diode-data"           % "1.1.7",
+    "io.suzaku"                         %%% "diode-react"          % "1.1.7.160"
```

Comment:
Why do we need `diode-react`?

Reply:
IIRC, that provides the `.render...` methods for `Pot`. The usage of `diode-*` is temporary, we are only using `Pot`. We should extract and adapt the needed logic.